.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
probably can gitignore the whole folder

Reply:
Uh no, that file is so people can easily run the example using vscode, because it needs assertions and such, this lowkey makes life a lot easier for contributors

Reply:
In general I think it is kind of frowned upon to check in editor specific files, but maybe there is enough of a percentage of vscode users in flutter for this to make sense?

Reply:
I mostly added it for contributors, it is not something that gets released to pub.dev. It just makes testing a lot easier. So not sure what would be smart in this case haha